packages/ai/cypress/specs/Input.cy.tsx

```diff
@@ -0,0 +1,950 @@
+            const onStopGeneration = cy.spy().as("onStopGeneration");
+
+            cy.mount(
+                <AIInput
+                    loading={true}
+                    onStopGeneration={onStopGeneration}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .realClick();
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("[ui5-icon]")
+                .realClick();
+
+            cy.get("@onStopGeneration").should("have.been.calledOnce");
+        });
+    });
+
+    describe("Keyboard Shortcuts", () => {
+        it("should handle Shift+F4 to open menu", () => {
+            cy.mount(
+                <AIInput>
+                    <MenuItem slot="default" text="Generate"></MenuItem>
+                </AIInput>
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Shift', 'F4']);
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("[ui5-menu]")
+                .ui5MenuOpened();
+        });
+
+        it("should handle Ctrl+Shift+Z for previous version when multiple versions exist", () => {
+            const onVersionChange = cy.spy().as("onVersionChange");
+
+            cy.mount(
+                <AIInput
+                    loading={false}
+                    currentVersion={2}
+                    totalVersions={3}
+                    onVersionChange={onVersionChange}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Control', 'Shift', 'Z']);
+
+            cy.get("@onVersionChange")
+                .should("have.been.calledOnce")
+                .its("firstCall.args.0.detail")
+                .should("deep.equal", {
+                    backwards: true
+                });
+        });
+
+        it("should handle Ctrl+Shift+Y for next version when multiple versions exist", () => {
+            const onVersionChange = cy.spy().as("onVersionChange");
+
+            cy.mount(
+                <AIInput
+                    loading={false}
+                    currentVersion={2}
+                    totalVersions={3}
+                    onVersionChange={onVersionChange}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Control', 'Shift', 'Y']);
+
+            cy.get("@onVersionChange")
+                .should("have.been.calledOnce")
+                .its("firstCall.args.0.detail")
+                .should("deep.equal", {
+                    backwards: false
+                });
+        });
+    });
+
+    describe("Input Integration", () => {

```

Comment:
I don't think we have to test the inherited behavior unless we are rewriting some ui5-input features/properties in the "child" component.